splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -76,6 +76,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1933), new UpgradeScriptToUpdateViewForSYSCONGLOMERATEINSCHEMAS(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1938), new UpgradeScriptForTriggerWhenClause(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1941), new UpgradeScriptForTableColumnViewInSYSIBM(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1946), new UpgradeScriptForAddDefaultToColumnViewInSYSIBM(sdd,tc));
```

Comment:
We need to update the build number

Reply:
Updated the build number to 1948 for all 3 branches, thanks!